lib/eth/EthRpc.js

```diff
@@ -204,8 +204,14 @@ class EthRPC {
     return this.web3.eth.personal.lockAccount(account);
   }
 
-  getTransaction({ txid }) {
-    return this.web3.eth.getTransaction(txid);
+  async getTransaction({ txid, getConfirmations = false }) {
+    const tx = await this.web3.eth.getTransaction(txid);
+    if (!getConfirmations || !tx) {
+      return tx;
+    }
+    const bestBlock = await this.web3.eth.getBlockNumber();
+    tx.confirmations = (tx.blockNumber && tx.blockNumber !== 0) ? bestBlock - tx.blockNumber + 1 : 0;
```

Comment:
I think this should just be 
`tx.blockNumber !== undefined ? bestBlock - tx.blockNumber + 1 : 0`

Reply:
fixed